components/Footer.tsx

```diff
@@ -0,0 +1,46 @@
+import Link from 'next/link';
+
+export default function Footer() {
+  return (
+    <footer className="footer">
+      <div className="gel-wrap">
+        <div className="footer__social">
+          <div className="footer__social__item">
+            <a
+              href="https://www.facebook.com/roseatecards"
+              target="_blank"
+              rel="noopener noreferrer"
```

Comment:
Think we need to add `rel="external"` to indicate that "the referenced document is not part of the same site as the current document" (https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel)

Reply:
Done.